# Define the goal
string tracking_frame
int32 tracking_time
bool stop_on_goal
---
# Define the result
bool success
---
# Define a feedback message
geometry_msgs/Twist twist
float64 distance
